### PR TITLE
[4.0]User - User Actions Log - Need Tooltip for search field #26659

### PR DESCRIPTION
**Pull request by sudhir-j-sapkal**
### Summary of Changes
Pull Request for Issue #[#26659 ] - User - User Actions Log - Need Tooltip for search field

### Testing Instructions
Go to Administrator Login Page
Click on Users -> User Actions Log
Check Search input box tooltip

### Expected result

After hovering on the search field tooltip should display.

### Actual result

Screenshot from 2019-10-19 16-35-26

### CODE REVIEW

**Comment by manojLondhe** [comment]
I have tested this item :white_check_mark: successfully on 2118fa0c4d73a62eb7996f7923f9d85d19c87595<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/26676">issues.joomla.org/tracker/joomla-cms/26676</a>.</sub>

**Comment by uthorat** [comment]
I have tested this item :white_check_mark: successfully on 2118fa0c4d73a62eb7996f7923f9d85d19c87595<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/26676">issues.joomla.org/tracker/joomla-cms/26676</a>.</sub>

File: administrator/language/en-GB/en-GB.com_actionlogs.ini [hunk 1]
```diff
@@ -51,6 +51,7 @@ COM_ACTIONLOGS_POSTINSTALL_TITLE="User Actions Can Now Be Logged"
 COM_ACTIONLOGS_PURGE_CONFIRM="Are you sure want to delete all User Action logs?"
 COM_ACTIONLOGS_PURGE_FAIL="Failed to delete all User Action logs."
 COM_ACTIONLOGS_PURGE_SUCCESS="All User Action logs have been deleted."
+COM_ACTIONLOGS_SEARCH_IN_NAME="Search in username. Prefix with ID: to search for a action log ID."
```
Comment (Quy) [inline administrator/language/en-GB/en-GB.com_actionlogs.ini]:
```suggestion
COM_ACTIONLOGS_SEARCH_IN_NAME="Search in username. Prefix with ID: to search for an action log ID."
```

Comment (sudhir-j-sapkal) [reply administrator/language/en-GB/en-GB.com_actionlogs.ini]: @Quy - DONE

**Comment by Quy** [comment]
RTC<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/26676">issues.joomla.org/tracker/joomla-cms/26676</a>.</sub>

**Comment by uthorat** [comment]
I have tested this item :white_check_mark: successfully on 6df0cfcb2d998415914b137d3dbf162b6389c33b<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/26676">issues.joomla.org/tracker/joomla-cms/26676</a>.</sub>

**Comment by kk9** [comment]
I have also tested successfully<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/26676">issues.joomla.org/tracker/joomla-cms/26676</a>.</sub>

**Comment by joeykrop** [comment]
I have tested this item :white_check_mark: successfully on 6df0cfcb2d998415914b137d3dbf162b6389c33b<br /><br />Works as expected.<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/26676">issues.joomla.org/tracker/joomla-cms/26676</a>.</sub>

**Comment by wilsonge** [comment]
Thanks!

**Pull request merged**